src/v/storage/tests/appender_chunk_manipulations_test.cc

```diff
@@ -31,35 +29,35 @@ SEASTAR_THREAD_TEST_CASE(chunk_manipulation) {
 
     {
         c.append(b.data(), c.space_left());
-        BOOST_REQUIRE(c.is_full());
-        BOOST_REQUIRE_EQUAL(c.size(), chunk_size);
+        ASSERT_TRUE(c.is_full());
+        ASSERT_EQ(c.size(), chunk_size);
         c.reset();
     }
     {
         c.append(b.data(), alignment - 2);
         c.append(b.data(), 4);
-        BOOST_REQUIRE_EQUAL(c.dma_size(), 8192);
+        ASSERT_EQ(c.dma_size(), 8192);
         c.reset();
     }
     {
         size_t i = (alignment * 3) + 10;
         c.append(b.data(), i);
-        BOOST_REQUIRE_EQUAL(c.dma_size(), alignment * 4);
+        ASSERT_EQ(c.dma_size(), alignment * 4);
         const char* dptr = c.dma_ptr();
         const char* eptr = b.data();
-        BOOST_REQUIRE(std::memcmp(dptr, eptr, i) == 0);
+        ASSERT_EQ(std::memcmp(dptr, eptr, i), 0);
         c.flush();
         // same after flush
-        BOOST_REQUIRE_EQUAL(c.dma_size(), alignment);
+        ASSERT_EQ(c.dma_size(), alignment);
         // 10 bytes on the next page
-        BOOST_REQUIRE_EQUAL(c.flushed_pos() % alignment, 10);
-        BOOST_TEST_MESSAGE("10 bytes spill over: " << c);
+        ASSERT_EQ(c.flushed_pos() % alignment, 10);
+        fmt::print("10 bytes spill over: {}\n", c);
```

Comment:
Rather than logging to stdout, wdyt about something like adding an RPTEST_INFO() test macro that wraps vlog with a test-only logger?